### ISSUE TITLE
Is VoLTE calling possible?

### ISSUE THREAD

**Issue by gadgetguy08** [SimpleMobileTools/Simple-Dialer#74]
Is it possible to get Voice over LTE (also known as HD Voice) with this app?
I didn't see a setting and it didn't kick into that mode when I made a test call.

**Comment by tibbi** [comment]
not really, it isnt possible. I dont think it will be added anytime soon. I havent researched it, not sure how to add it.

**Comment by mareking** [comment]
@gadgetguy08 Very good questions and suggestion! I did not think about it before but this could really render Simple Dialer unusable in the future... Globally more and more mobile network operators are shutting down 2G and 3G networks. So VoLTE will not only be the best choice quality-wise but often the only option to make a regular phone call in the future.

@tibbi I do not know of any other custom or even FOSS dialer yet that supports VoLTE. Hopefully you can find one, I will let you know if I do.

**Comment by naveenjohnsonv** [comment]
My ISP only supports VOLTE calls and it works just fine for me

**Comment by genodeftest** [comment]
@naveenjohnsonv wrote:
> My ISP only supports VoLTE calls and it works just fine for me

Do you meant with Simple-Dialer? Or with the stock android phone app?

In the first case, this would be handled by the operating system.

**Comment by naveenjohnsonv** [comment]
> @naveenjohnsonv wrote:
> 
> > My ISP only supports VoLTE calls and it works just fine for me
> 
> Do you meant with Simple-Dialer? Or with the stock android phone app?
> 
> In the first case, this would be handled by the operating system.

Yeah, Simple-Dialer. It should be handled by the operating system, while having the app's calling interface instead of the stock android app's. So I don't understand why this app specifically needs to add any support for it

**Comment by Krawei** [comment]
@naveenjohnsonv How do you proof that simple dailer is using VoLTE? I don't see any icon or kind of label when calling someone or after a call in the call log. 
@tibbi is it possible to implement kind of icon in the call log?Like for instance Xiaomi does in their phone app?

**Comment by naveenjohnsonv** [comment]
> @naveenjohnsonv How do you proof that simple dailer is using VoLTE? I don't see any icon or kind of label when calling someone or after a call in the call log.
> @tibbi is it possible to implement kind of icon in the call log?Like for instance Xiaomi does in their phone app?

Because my carrier only supports VoLTE calls. So if a call goes through then it's through the LTE band.
